### ISSUE TITLE
[Feature Request]: API for Apps, Channels and Schemes

### ISSUE THREAD

**Issue by savage7** [tryzealot/zealot#1485]
### 描述 | Description

Hi @icyleaf 

first off: zealot is a really cool project, thank you for your great work 🥇 
I'm currently looking for an AppCenter Alternative and zealot stands out with most of its features. 

I want to automate the creation of apps, schemes and channels in the CI. The idea is to automatically create an app and a set of schemes&channels based on the bundle/package id. 

I have something in mind like this:
Get the list of apps and check if the app, scheme & channel has not been created via API. If not exists create them via API:
- Create an app i.e. based on the bundle/packageid of an app via API
- Create the scheme of an app via API
- Create a channel in a scheme for an app via API
- Upload build via fastlane with the channel id from the created channel (or the one we got via the list of apps)

Basically a CRUD API for Apps, Schemes and Channels. This way we would be very flexible to handle app creation in the CI. :heart: 

Is this something you're planning?

**Comment by icyleaf** [comment]
Very clear requirements, the workload is not too heavy, I will arrange it next.

**Comment by savage7** [comment]
@icyleaf when do plan to release this? Seams to be implemented on dev for a while now 😃

**Comment by icyleaf** [comment]
I am considering releasing the stable version after completing #1496. If the first deployment can use the `nightly` version, I have been using this version for a while without any major issues.

**Comment by savage7** [comment]
Thanks you!

**Comment by icyleaf** [comment]
The version 5.3.0 will be released ASAP.

**Comment by savage7** [comment]
Thanks alot, I'm using the nightly successfully since last week. Didn't have any issue since 🥇
